build/gke-test-cluster/cluster-e2e.yml

```diff
@@ -19,10 +19,10 @@ resources:
     cluster:
       name: e2e-test-cluster
       description: End to end tests cluster for Agones
-      initialClusterVersion: "1.11"
+      initialClusterVersion: "1.12"
       nodePools:
         - name: "default"
-          initialNodeCount: 4
+          initialNodeCount: 8
```

Comment:
8*4 = 32 which i believe is larger than the default quota in a zone for new projects. Do we need to bump this for individuals to run e2e tests? I know the shared e2e cluster has 8 nodes, but in my very limited testing with my own e2e cluster 4 seemed sufficient.

Reply:
So this is for the agones-images project, not really for individuals (although they are welcome to use it too). We run a higher parallel rate (32 from memory), and need the extra room.

In fact, given that e2e now takes 8 minutes, we could bump that up a bit too - hence the extra space.

Reply:
Ah, I didn't realize that this was only used for e2e. The e2e cluster already has 8 nodes, so this just makes that consistent.